terminal/tcell/tcell.go

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Google Inc.
```

Comment:
Probably missed from the previous round, posting as a standalone comment so we an discuss:

I am wondering if we should also update the termdashdemo. We could add a command line argument that would allow running it optionally with tcell for an easy comparison. What do you think? This would also allow me to easily test and verify functionality of the tcell layer.

Reply:
Would we only want to update termdashdemo? What about the other widget demos?

Reply:
I am 👍  for adding a flag to switch the implementation.

Reply:
That is a good question, I think eventually we will want to add this to all the demos and since tcell does seem better and assuming no negative performance implications - we might as well make it the default.

However we don't need to do it in this PR, it would just extend its scope. I would still go for termdashdemo though, so that we can play around with tcell a bit and maybe also profile the two and see differences in CPU and memory usage.